BAZProject/SearchMovies/SearchViewModel.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  SearchViewModel.swift
+//  BAZProject
+//
+//  Created by Manuel S on 18/04/23.
+//
+
+import Foundation
+
+class SearchViewModel {
+    
+    private var service: Service
+    
+    init(service: Service = ServiceAPI(session: URLSession.shared)) {
+        self.service = service
+    }
+    
+    //MARK:Funciones para la vista 
```

Comment:
Cuando agregas un guión corto después de tu MARK, este organizara lo que haya después del mark, hasta encontrar otro, además de agregar una tenue línea en el editor que divide lo que hay ahí.

  ```suggestion
    // MARK: - Funciones para la vista 
```
![Screen Shot 2023-04-24 at 15 38 09](https://user-images.githubusercontent.com/85137047/234122464-da491b04-3360-46e3-9d24-eb771d90e6cf.png)
![Screen Shot 2023-04-24 at 15 38 00](https://user-images.githubusercontent.com/85137047/234122467-5ced709f-8503-42df-aa39-336edec41729.png)